.github/scripts/fetch_latest_green_commit.py

```diff
@@ -39,22 +39,33 @@ def get_latest_commits() -> List[str]:
 
     return commits
 
-def query_commits(commits: List[str], qlambda: Any) -> Any:
-    params = rockset.ParamDict()
-    params['shas'] = ",".join(commits)
-    results = qlambda.execute(parameters=params)
+def query_commits(commits: List[str]) -> List[Dict[str, Any]]:
+    rs = rockset.RocksetClient(
+        host="api.usw2a1.rockset.com", api_key=os.environ["ROCKSET_API_KEY"]
```

Comment:
Different endpoints for the new Rockset API?

Reply:
not entirely sure, but followed https://github.com/rockset/rockset-python-client#migration-guide and https://rockset.com/docs/rest-api/ to make these changes